single-line-diagram-core/src/main/java/com/powsybl/sld/model/SwitchNode.java

```diff
@@ -50,5 +50,11 @@ public Node getOtherAdjNode(Node adj) {
     protected void writeJsonContent(JsonGenerator generator, boolean includeCoordinates) throws IOException {
         super.writeJsonContent(generator, includeCoordinates);
         generator.writeStringField("kind", kind.name());
+        if (this.getOrder().isPresent()) {
```

Comment:
you can remove the `this.` keyword in the added lines to slightly reduce the code